src/components/Articles/CompanyProvisions.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <div id="company-provisions">
+    <confirm-dialog ref="confirmCompanyProvisionsDialog" attach="#company-provisions" />
+    <v-layout v-if="!isEditing">
+      <v-flex xs3>
+        <label><strong>Pre-existing Company Provisions</strong></label>
+        <v-chip v-if="hasProvisionsRemovedPropsChanged" x-small label color="#1669BB" text-color="white">
+          CHANGED
+        </v-chip>
+      </v-flex>
+      <v-flex xs7 class="info-text" v-if="provisionsRemoved">
+        The company has resolved that none of the Pre-existing Company Provisions are to apply to this company.
+      </v-flex>
+      <v-flex xs7 class="info-text" v-else>
+        This company has Pre-existing Company Provisions
```

Comment:
Since alterate sentence ends in a period, this one should too.

Reply:
Will change.

Reply:
Done.